packages/charting/src/components/ProgressBarChart/examples/ProgressBarChart.Basic.Example.tsx

```diff
@@ -0,0 +1,18 @@
+import * as React from 'react';
+
+import { customizable } from 'office-ui-fabric-react/lib/Utilities';
+import { ProgressBarChart, IProgressBarChartProps } from '@uifabric/charting/lib/ProgressBarChart';
+
+@customizable('ProgressBarChartBasicExample', ['theme', 'styles'])
+export class ProgressBarChartBasicExample extends React.Component<IProgressBarChartProps, {}> {
+  public render(): JSX.Element {
+    return <div>{this._basicExample()}</div>;
+  }
+
+  private _basicExample(): JSX.Element {
+    const points = { value: 500, total: 1000 };
+    const colors = { backgroundColor: 'grey', barColor: 'blue' };
```

Comment:
Let's use colors from the design here so the example is something a team can copy/paste. Recommend:
`{ backgroundColor: '#c4c4c4', barColor: '#0078d7'};`